ndmg/stats/qa_reg.py

```diff
@@ -90,9 +89,25 @@ def opaque_colorscale(basemap, reference, vmin=None, vmax=None, alpha=1):
     cmap[:, :, 3] = opaque_scale
     return cmap
 
+def pad_im(image,max_dim,pad_val):
+    """
+    Pads an image to be same dimensions as given max_dim
+    Parameters
+    -----------
+    image: 3-d RGB np array of image slice
+    max_dim: dimension to pad up to
+    pad_val: value to pad with
+    Returns
+    -----------
+    padded_image: 3-d RGB np array of image slice with padding
+    """
+    #pad only in first two dimensions not in rgb
+    pad_width = (((max_dim-image.shape[0])//2,(max_dim-image.shape[0])//2),((max_dim-image.shape[1])//2,(max_dim-image.shape[1])//2),((max_dim-image.shape[2])//2,(max_dim-image.shape[2])//2))
+    padded_image = np.pad(image, pad_width=pad_width, mode='constant', constant_values=pad_val)
+    return padded_image
```

Comment:
another example of code that's repeated across PRs. Should be defined only once in a different module, and then imported.